Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,147 @@
     "\n",
-    "### Within Text Body\n",
-    "Use single `$ <equation> $` symbol to put and equation (e.g. ```$y = \\delta ~,~ u = 0.99 ~U_e .$```) within the text body.\n",
+    "<img src=\"./Images/Lift_plot_combined.jpg\" width=\"900\">\n",
+    "Fig. 4. Temporal variation of $C_L$ for flow over a rotating cylinder with three different rotational speed.\n",
     "\n",
-    "Use single `$ <equation> $` symbol to put and equation (e.g. $y = \\delta ~,~ u = 0.99 ~U_e .$) within the text body.\n",
+    "Once the oscillatory behavior of the flow field as a function of time reaches a steady manner users should quantify the flow field variables by looking at the mean value of each variable and neglecting their redundant fluctuations. The mean values of lift $C_L$ and drag $C_D$ force coefficients are calculated and presented in Fig. 2 as a function of rotational speed of the cylinder. As shown in Fig. 2 with the increases in the rotational speed of the cylinder $\\alpha$, the value of drag coefficient decreases. This is mainly due to the reduction of pressure gradient across the cylinder due to rotation and strong mixing phenomena in the local flow. On the other side of the spectrum as the cylinder rotational rate $\\alpha$ increases, the value of lift coefficient $C_L$ magnitude increases in negative direction. \n",
```

Comment:
You should elaborate on the procedure to calculate and report the mean. Again this is not a paper is tutorial/report.